kpatch-build/insn/inat-tables.c

```diff
@@ -2,7 +2,7 @@
 /* Do not change this code. */
 
 /* Table: one byte opcode */
-const insn_attr_t inat_primary_table[INAT_OPCODE_TABLE_SIZE] = {
+static const insn_attr_t inat_primary_table[INAT_OPCODE_TABLE_SIZE] = {
```

Comment:
Dropping inat-tables.c from this PR sounds good and adding new_bugs.sh (now or later, if needed) is fine.

I agree that fixing inat-tables.c upstream would be the best long term approach.  (If you wanted to do that yourself, be aware that there are three copies of that code in the kernel tree which are all kept in sync, so they would need to be updated simultaneously.  I own one of those copies (objtool), my eventual goal is to share the code with perf so that there are only two copies in the tree, but it's way down on my TODO list.)